kotlinpoet/src/test/java/com/squareup/kotlinpoet/MemberNameTest.kt

```diff
@@ -320,4 +320,35 @@ class MemberNameTest {
       |}
       |""".trimMargin())
   }
+
+  @Test fun importOperator() {
+    val rangeTo = MemberName("com.squareup.tacos", "rangeTo")
+    val not = MemberName("com.squareup.tacos", "not")
+    val compareTo = MemberName("com.squareup.tacos", "compareTo", ">=")
+    val equals = MemberName("com.squareup.tacos", "equals")
+    val funSpec = FunSpec.builder("makeTastyTacos")
+        .addStatement("%OAny()%OAny()", not, rangeTo)
+        .addStatement("Any() %O Any()", compareTo)
+        .addStatement("Any() %O Any()", equals)
+        .addStatement("Any() %O Any()", equals.copy("!="))
+        .build()
+    val file = FileSpec.builder("com.example", "Tacos")
+        .addFunction(funSpec)
+        .build()
+    assertThat(file.toString()).isEqualTo("""
+      |package com.example
+      |
+      |import com.squareup.tacos.compareTo
+      |import com.squareup.tacos.equals
+      |import com.squareup.tacos.not
+      |import com.squareup.tacos.rangeTo
+      |
+      |fun makeTastyTacos() {
+      |  !Any()..Any()
```

Comment:
`!Any()..Any()` doesn't make much sense to me, something like `for (i in 0..10)` would be much easier to understand. Unit tests are part of KotlinPoet's documentation that we expect people to read, so writing them in the form of recipes that somebody can just copy into their codebase is great.